### PR TITLE
Bellman-Ford

### PR DESCRIPTION
**Pull request by rzarno**
Bellman-Ford - search graph shortest path algorithm implementation in PHP

### CODE REVIEW

**Review by darwinz — COMMENTED** [review]
Please add a reference to this algorithm/data structure in Directory.md

File: Graphs/BellmanFord.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    }
+                    $vertices[$edge->end] = $vertices[$vertice] + $edge->weight;
+                    $change = true;
+                }
+            }
+        }
+        $round++;
+    }
+    return $vertices;
+}
```
Comment (darwinz) [inline Graphs/BellmanFord.php:52]: The PHP linter wants a newline at the end of this file.

```
52 | ERROR | [x] Expected 1 newline at end of file; 0 found
```

**Comment by rzarno** [comment]
@darwinz can we merge this PR or I need to add/change sth?

**Review by darwinz — APPROVED** [review]
This looks great.  Thank you!

**Pull request merged**